packages/liveblocks-react-ui/src/components/internal/AiChatComposer.tsx

```diff
@@ -79,6 +80,13 @@ export interface AiChatComposerProps extends ComponentProps<"form"> {
    * The id of the copilot to use to send the message.
    */
   copilotId?: CopilotId;
+  /**
+   * The contextual knowledge to include in the chat. May be used by the
+   * assistant when generating responses. In addition to the knowledge passed
+   * in via this prop, the AiChat instance will also have access to any
+   * globally registered knowledge via <RegisterAiKnowledge />.
+   */
+  knowledge?: AiKnowledgeSource[];
```

Comment:
To me, it's a little strange that we don't have to pass tools to the composer this way but we do for knowledge. Relates to this my other [comment](https://github.com/liveblocks/liveblocks/pull/2570#issuecomment-3154904726).